s6/debian-root/etc/services.d/lighttpd/run

```diff
@@ -2,11 +2,23 @@
 
 s6-echo "Starting lighttpd"
 
-# Touch log files to ensure they exist (create if non-existing, preserve if existing)
-touch /var/log/lighttpd/access.log /var/log/lighttpd/error.log
+if [[ 1 -eq ${WEBLOGS_STDOUT:-0} ]]; then
+  #lighthttpd cannot use /dev/stdout https://redmine.lighttpd.net/issues/2731
+  rm -f /var/log/lighttpd/access.log
+  mkfifo -m 600 /var/log/lighttpd/access.log
+  rm -f /var/log/lighttpd/error.log
+  mkfifo -m 600 /var/log/lighttpd/error.log
+  chown -R www-data:www-data /var/log/lighttpd
+  service lighttpd-access-log start
```

Comment:
I'm still unsure of the best way to handle these.

Just for fun, exec into to the container and manually stop/start these services. Stopping does nothing and the two `cat` processes stay alive. Starting them again adds n number of additional `cat` processes.

Also restarting just the `lighttpd` service causes a loop:

![image](https://user-images.githubusercontent.com/1998970/151251348-40083f00-2d1b-458f-98f5-4ed68571ceba.png)

Reply:
adding these lines into lighttpd's finish fixes the problem.
```
service lighttpd-access-log stop
service lighttpd-error-log stop
killall -9 cat
```